src/services/index.js

```diff
@@ -23,7 +23,8 @@ const getVersion = async (env) => {
 const getVersions = async () => {
   const liveVersion = await getVersion('live');
   const testVersion = await getVersion('sandbox');
```

Comment:
Use Promise.all for faster responses
 
```suggestion
  const promises= [getVersion('sandbox'), getVersion('live')];
  const [sandboxResponse, LiveResponse] = await Promise.all(promises);
```